src/contentScript.js

```diff
@@ -219,37 +221,14 @@ var __gcrExtAnswers;
-
-    var closeBar = dialog.querySelector('.gcr-ext-editor-close');
-
-    var closeText = createNodeWithClass('span', 'select-menu-title');
-    closeText.innerHTML = 'Edit or add canned responses';
-    closeText.style.float = 'left';
-    closeText.style.padding = '5px 10px';
-    closeText.style.color = 'black';
-    closeText.style.fontWeight = 'bold';
-
-    var closeButton = createNodeWithClass('button', 'btn-link delete-button');
-    closeButton.type = 'button';
-    closeButton.style.padding = '5px 10px';
-    closeButton.style.float = 'right';
-    var svg = createSVG(16, 16, 'octicon-x', 'M7.48 8l3.75 3.75-1.48 1.48-3.75-3.75-3.75 3.75-1.48-1.48 3.75-3.75L0.77 4.25l1.48-1.48 3.75 3.75 3.75-3.75 1.48 1.48-3.75 3.75z');
-    closeButton.appendChild(svg);
-    closeButton.addEventListener('click', function() {
-      document.body.removeChild(dialog);
-    });
-
-    closeBar.appendChild(closeText);
-    closeBar.appendChild(closeButton);
+    dialog.innerHTML = '<h2 class="facebox-header">Edit or add canned responses</h2><div class="gcr-ext-editor-header"> <div class="gcr-ext-editor-horizontal"> <div> <input data-facebox-id="gcrExtNewTitle" class="gcr-ext-editor-answer-title gcr-ext-editor-answer-input" placeholder="You go get \'em tiger!"><br> <textarea data-facebox-id="gcrExtNewText" aria-label="Canned reply answer" class="gcr-ext-editor-answer-input" placeholder="You\'re the best! Also, we\'re closing this PR because it\'s written wrong, but <333"></textarea> </div> <div> <div class="gcr-ext-editor-answer-text"><span class="gcr-ext-editor-pink">⇠</span> This is an easy title to remember this canned response by<br><br><span class="gcr-ext-editor-pink">⇠</span> And this is the actual content that will be inserted<br><br> <button data-facebox-id="gcrExtNewButton" class="btn btn-sm btn-primary">Can it!</button> <span data-facebox-id="gcrExtNewError" class="gcr-ext-editor-status-message" hidden>No empty canned responses!</span> <span data-facebox-id="gcrExtNewConfirm" class="gcr-ext-editor-status-message" hidden>Added!</span> </div> </div> </div> </div> <div class="gcr-ext-editor-list"> <ul class="gcr-ext-answer-list"></ul> </div>';
```

Comment:
Because facebox is _coyping_ content, IDs get duplicated, and `data-facebox-id` is the way around it. When content gets copied, all the `data-facebox-id`s are turned into `id`s.